terraform-sentinel/step3.md

```diff
@@ -0,0 +1,32 @@
+In this step, you will edit your mock data structure to create a failing scenario based on the requirements in the policy. The passing test requires two tags and only allows for specific ACLs, so a failing test will have neither of those tags and have an ACL outside the bounds of your policy.
+
+# Open the failing mock scenario for editing
+
+There is a file called `mock-data/mock-tfplan-fail-v2.sentinel`{{open}}. Open that file and edit the values for your failing test.
+
+# Edit your mock data ACL
+
+At line 123, the mock contains the ACL for the configuration. Instead of `public-read`, which is allowed in your policy, change this to `public-read-write`. 
+
+If you are having trouble finding the specific line in the collection, search for `<UPDATE_VALUE>` in the file.
```

Comment:
Should they search for the exact string `<UPDATE_VALUE>` or is this a reference to another value like an ENV var?